scst-store/multicluster-setup.hbs.md

```diff
@@ -64,24 +58,13 @@ data:
 EOF
 ```
 
-### <a id='copy-amr'></a>Copy AMR CloudEvent Handler CA certificate from the View cluster
+### <a id='copy-amr'></a>Copy AMR CloudEvent Handler CA certificate data from the View cluster
```

Comment:
⛏️I feel the word data in this situation is extraneous which makes the phrase read a bit strange. I think think it's needed.